src/components/Header/__test__/Header.test.js

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+import { HashRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../../../redux/ConfigureStore';
+import Header from '../Header';
+
+describe('Test header', () => {
+  it('test if the heander is rendered correctly', () => {
+    const header = renderer
+      .create(
+        <Provider store={store}>
+          <Router>
+            <Header />
+          </Router>
+        </Provider>,
+      )
+      .toJSON();
+    expect(header).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly make sure you also include some unit tests for pure functions in your Redux code and integration tests using React Testing Library as required for the project. Right, you only tested some components.
Here are your **tests requirements** from the project's requirements:

![Screenshot from 2022-08-11 05-18-07](https://user-images.githubusercontent.com/83042742/184059544-e583515d-9aec-4ace-8ad2-37da1574b3da.png)